indico/modules/rb_new/forms/__init__.py

```diff
@@ -0,0 +1,17 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2019 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
```

Comment:
leave empty and again a case where we probably don't need a package

Reply:
btw, maybe add `indico/modules/rb_new/event/...` and put controllers.py, forms.py, fields.py, etc. in there? that way all the event stuff is grouped together. basically a submodule of rb_new ;)